examples/with-wallet-stamper/src/app/layout.tsx

```diff
@@ -1,25 +1,36 @@
+"use client";
+
 import type { Metadata } from "next";
 import { Inter } from "next/font/google";
 import "./globals.css";
 import { Providers } from "@/components/providers";
 
+import { TurnkeyProvider as TurnkeyReactProvider } from "@turnkey/sdk-react";
+
 const inter = Inter({ subsets: ["latin"] });
 
 interface RootLayoutProps {
   children: React.ReactNode;
 }
 
-export const metadata: Metadata = {
+const metadata: Metadata = {
   title: "Turnkey Wallet Stamper Demo",
   description: "Demonstrates the usage of the @turnkey/wallet-stamper package",
 };
 
+const turnkeyConfig = {
+  apiBaseUrl: process.env.NEXT_PUBLIC_BASE_URL!,
+  defaultOrganizationId: process.env.NEXT_PUBLIC_ORGANIZATION_ID!,
+};
+
 export default function RootLayout({ children }: RootLayoutProps) {
   return (
     <html lang="en" suppressHydrationWarning className="dark">
-      <body className={inter.className}>
-        <Providers>{children}</Providers>
-      </body>
+      <TurnkeyReactProvider config={turnkeyConfig}>
```

Comment:
We essentially have two wrappers here: `TurnkeyReactProvider` from `@turnkey/sdk-react`, and another one that's created within this project. I included `TurnkeyReactProvider` in this demo because it abstracts away some iframe-related logic, so if it's confusing as to why where are two separate providers, I hope that offers some clarity.